test/run_tests.sh

```diff
@@ -63,8 +66,23 @@ for gemfile in "${gemfiles[@]}" ; do
   [[ $status -ne 0 ]] && echo "!!! Test suite failed for $gemfile with Ruby $ruby_version !!!"
 done
 
+# explicitly test for solarwinds initialization
+if ! BUNDLE_GEMFILE=gemfiles/test_gems.gemfile bundle update; then
+  echo "Problem during gem install. Skipping tests for $gemfile"
+  exit_status=1
+  continue
+fi
+
+for i in {1..7}; do
```

Comment:
It's hard because it will execute under same ruby environment which is something I want to avoid. AFAIK, it's hard to uninitialized initialized object in ruby. So I would like to keep the for loop; but I set bash to auto-detect the number of file so no hard-coded number there